### ISSUE TITLE
Alkyoneus randomly despawned while we were clearing a path to him.

### ISSUE THREAD

**Issue by gweivyth** [EdenServer/community#90]
Not sure why he would despawn.  He's just a normal lottery pop on retail in era.  We made sure to check downstairs in case he pathed down a hole, but he wasn't there either.

**Comment by Aneyus** [comment]
He was a force pop starting in 2007. Referenced in this thread here https://www.bluegartr.com/threads/43691-3-07-Patch-Details

Should we consider if we want to revert it to being popped by Moldy Bucklers or leave it as lottery first?

**Comment by jmcmorris** [comment]
This change was made for RMT reasons iirc. So we probably want to have it be lottery, right?

**Comment by godmode** [comment]
He’s 21/24 and I thought I had written a comment here. I know what the issue is and it’s an easy fix so I’ll do it when I get back.

**Comment by Aneyus** [comment]
Not sure if this issue is fixed. Alkyoneus might still be depopping.

**Comment by godmode** [comment]
He shouldn’t be but let me know if he is

**Comment by Timonthy** [comment]
Hi! Related - this has happened twice now, today and a week or so ago. I pulled Pallas on a THF to the lower level, then used Hide. All links despawned except Pallas, who lingered for a few seconds, warped back up through the ceiling, lingered a few more, then despawned.

His bats seemed to still be up, I tested this by pulling them, linking others, and then hiding again. All the other bats despawned except "his", they slowly walked back up to the spot I pulled them from and then despawned themselves.
